.github/workflows/stale.yml

```diff
@@ -31,11 +31,10 @@ jobs:
             Hi everyone! This issue has been closed due to inactivity.
             If you think this issue is still relevant in the latest Solidity version and you have something to [contribute](https://docs.soliditylang.org/en/latest/contributing.html), feel free to reopen.
             However, unless the issue is a concrete proposal that can be implemented, we recommend starting a language discussion on the [forum](https://forum.soliditylang.org) instead.
-          any-of-issue-labels: stale # TODO: remove this when we're done with closing ancient issues
-          ascending: true # TODO: remove this when we're done with closing ancient issues
+          ascending: true
```

Comment:
So, are we done with closing ancient issues? We already had a period where `debug` was false`, and all issues 334 days or older were immediately closed, so should this be removed now?

Reply:
Yes, we are done with them, but we will prioritize the oldest ones first. See this comment: https://github.com/ethereum/solidity/pull/13947#discussion_r1101901097